docs/configuration/config-file-reference.md

```diff
@@ -69,6 +69,12 @@ Where default_value is the value to use if the environment variable is undefined
 [http_prefix: <string> | default = "/api/prom"]
 
 api:
+  # Use compression for the Ruler and Alertmanager configuration API responses.
+  # Some endpoints of these components serve large YAML or JSON blobs which can
+  # benefit from compression.
+  # CLI flag: -api.enable-response-compression
+  [enable_response_compression: <boolean> | default = false]
```

Comment:
Would you mind to call it `response_compression_enabled` (and rename CLI flag accordingly) to keep consistency with the new bool naming? In the past we had no consistency, but now we're trying to name it `*_enabled`.

Reply:
Good shout, I tried to mimic other flags we had in the ruler. I'll always err on the side of consistency.